### PR TITLE
[Week-2]: adjust slides

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Remove type casting/assertions

## How can we measure success?

Less confusion

## Have we considered potential risks?

Poeple might see this in their codebase, so we need to address at some later session

### CODE REVIEW

File: 2-primitives/slides.md [hunk 1]
```diff
@@ -246,39 +214,201 @@ albumName;
...
+
+### Benefits of Using Records
+
+- Ensures data consistency by defining expected keys and value types.
+- Makes it easier to manage and work with structured data.
+- Offers better type checking and improved code reliability.
+
+---
+
+### Task: Managing Newspaper Articles
```
Comment (mxdvl) [inline 2-primitives/slides.md:382]: For the tasks, would it not be simpler to just point to actual `task.ts` files? Not sure these work well as slides.

We can alternatively create links to the TypeScript playground.

File: 2-primitives/lesson-objectives.MD [hunk 2]
```diff
@@ -1 +1,19 @@
...
+
+- Learn how to annotate types and create type aliases for code clarity.
+
+Type Safety & Literals:
+
+- Grasp type safety with type guards and apply literal types for specific values.
+
+Advanced Primitives:
+
+- Explore advanced primitive types and their practical applications.
```
Comment (mxdvl) [inline 2-primitives/lesson-objectives.MD:19]: Not a fan of the word “advanced”

**Review by dskamiotis — APPROVED** [review]

File: 2-primitives/lesson-objectives.MD (hunk 2 above)
Comment (dskamiotis) [reply 2-primitives/lesson-objectives.MD:19]: ah yes hold ill change

File: 2-primitives/slides.md (hunk 1 above)
Comment (dskamiotis) [reply 2-primitives/slides.md:465]: Good idea- ill keep the short snippets and add the more complex / longer ones to a .ts file 👍

**Pull request merged**